server/package.json

```diff
@@ -1,4 +1,29 @@
 {
   "name": "server",
-  "main": "server.js"
+  "main": "server.js",
+  "restApiRoot": "/api",
```

Comment:
Hmm, this does not look right to me. I believe server config should go to `server/config.json`, not `server/package.json`?